src/bindings/python/flux/core/handle.py

```diff
@@ -55,6 +55,11 @@ def __init__(self, url=ffi.NULL, flags=0, handle=None):
             destructor=raw.flux_close,
```

Comment:
(commit description) "parameter" is misspelled 2x.

Reply:
Thanks! Rebased and pushed with a fix for the typos in that commit message.